classes.py

```diff
@@ -0,0 +1,15 @@
+import requests
+from urllib.parse import urljoin
+
+class JokeApi:
+
+    base_url = "https://official-joke-api.appspot.com/"
+
+    @classmethod
+    def get_random_joke(cls):
+
+        endpoint = "random_joke"
+        url = urljoin(cls.base_url, endpoint)
+        response = requests.get(url)
+        
+        return response.json()
```

Comment:
Nice work! I would probably call this file `clients.py` as that is what you call objects created for the sole purpose of talking to an external service, in this case the joke api :)

Reply:
Will do, thanks!